Client/mods/deathmatch/logic/luadefs/CLuaBuildingDefs.h

```diff
@@ -19,7 +19,7 @@ class CLuaBuildingDefs : public CLuaDefs
     static void AddClass(lua_State* luaVM);
 
     // Buiding create funcs
-    static CClientBuilding* CreateBuilding(lua_State* const luaVM, uint16_t modelId, CVector pos, CVector rot, std::optional<uint8_t> interior);
+    static CClientBuilding* CreateBuilding(lua_State* const luaVM, uint16_t modelId, CVector pos, std::optional<CVector> rot, std::optional<uint8_t> interior);
```

Comment:
According to the encoding guidelines, std::uint16_t, std::uint8_t, etc. should be used.

Reply:
oh, this is official now ;(
weird

Reply:
Why did you mark the conversations as resolved if you didn't make any changes?

Reply:
My bad. I didn't push changes

Reply:
yeah, my local branch was conflicting with remote